testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/iioimage/IIOImageProviderTest.java

```diff
@@ -40,7 +39,8 @@
 @RunAsClient
 public class IIOImageProviderTest {
    static ResteasyClient client;
-   static final String testPngResource = "test.png";
+   static final String testPngResource1 = "test1.png";

```

Comment:
Just a nit, can the filenames or names of the properties suggest why different image files are checked? Something like "test_image_jdk8.png" and "test_image_jdk9plus.png" and/or comment, WDYT?

Reply:
added a comment, ok